membership-attribute-service/app/services/subscription/ZuoraSubscriptionService.scala

```diff
@@ -22,8 +21,9 @@ import utils.SimpleEitherT.SimpleEitherT
 import scala.concurrent.{ExecutionContext, Future}
 import scala.language.higherKinds
 import scala.util.Try
+import _root_.services.zuora.rest.SimpleClient
```

Comment:
This import looks odd, does it need the _root_ bit?

Reply:
Overall it's needed in certain situations - like when there is a package on top level with the same name as subpackage (of the package of class the import sits in) - scala then can't reconcile if we mean relative or absolute.
Also sometimes needed when we wildcard-imported some package.

Reply:
In this case there's
import com.gu.memsub.subsv2._
higher up and subsv2 package has services subpackage.

At some point I'll restructure the packages so we won't have this problem.